api/src/orgs/schema.js

```diff
@@ -10,6 +10,8 @@ export default `
     kit: String
     # The total amount of assets under management (in Dai) for the organisation.
     aum: Float
+    # The total amount of ANT for the organisation.
+    ant: Float
```

Comment:
Not sure if we want to expose `ANT` in the API, but it already appears in the [resolver](https://github.com/1Hive/apiary/blob/master/api/src/orgs/resolver.js#L23).

Reply:
Can you make this a separate PR? I think I missed it in #144

Reply:
It solves:
`UnhandledPromiseRejectionWarning: Error: Organisation.ant defined
in resolvers, but not in schema`

Which is not really related with this pull request, but isn't related with #144 as it is a bug by itself.

Do you want me to do a separate pull request with only that fix?

Reply:
I moved the second commit to #150.